### PR TITLE
Basic fake SMMT service

### PR DESCRIPTION
**Pull request by Marcin-Duszynski**
Basic fake SMMT service without deployment to AWS and without reading api key from AWS environment variable.

Usable only on local development environment.

How to start:
* cd /app
* npm install
* npm start

How to execute unit tests:
* cd /app
* npm test

### CODE REVIEW

**Comment by przemos** [comment]
We need to talk about using eslint to enforce coding standards

**Comment by Marcin-Duszynski** [comment]
esLint added. Temporary standard coding guidelines are used. Tomorrow we need to discuss with the team final code style rules.

File: app/test/vehicles.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+let chai = require('chai')
+let should = chai.should()
+
+let vehicles = require('../vehicles')
+
+describe('When checking if vehicle has a outstanding recall', () => {
+  describe('and valid VIN and MARQUE is provided', () => {
+    it('and user provide upper case text and vehicle has a recall and "Recall Outstanding" message is provided.', () => {
```
Comment (ryszardm-kainos) [inline app/test/vehicles.js]: Think the reporting description won't be descriptive this way. Usually the standard way for units is to describe what we are testing in `describe` and outlining the expected behaviour in `it`.  So if those are unit tests we could perhaps
```
describe('vehicles' (...)
   describe('getRecall' (...)
      it('behaviour' (...)
```

What do you think?

Comment (Marcin-Duszynski) [reply app/test/vehicles.js]: Discussed f2f

**Pull request merged**